.github/kokoro/db-full.sh

```diff
@@ -69,37 +60,9 @@ echo "----------------------------------------"
 		rm $tmp
 		exit $DATABASE_RET
 	fi
-
```

Comment:
Why are you removing the check which makes sure that there is nothing left to run after a complete fuzzing run?

Reply:
Because the check is wrong when there are multiple sub-directories.  The Makefile refactoring resulted in multiple make sub-direction recursions, which this code cannot handle.  There is not a straight forward replacement that isn't fragile, like this previous code was.

Reply:
Shouldn't `make --dry-run db-${XRAY_SETTINGS}-all` check that the `make db-${XRAY_SETTINGS}-all` target doesn't have anything left to do? Or is `db-${XRAY_SETTINGS}-all` a phony target so `make` always returns something when using `--dry-run`?

Reply:
> Or is `db-${XRAY_SETTINGS}-all` a phony target so `make` always returns something when using `--dry-run`?

This, sort of.  Each leaf phony target reports "nothing to do", but there are bunch of "entering directory", "leaving directory" and the print of the recursive make target invocation.  The end result is it is fairly hard to parse the output.